extensions/arc/package.json

```diff
@@ -738,26 +738,75 @@
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_ENGINE_VERSION",
+                          "type": "options",
+                          "options" : ["11","12"],
+                          "defaultValue": "12"
+                        },
+                        {
+                          "label": "%arc.postgres.server.group.extensions.label%",
+                          "description": "%arc.postgres.server.group.extensions.description%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_EXTENSIONS",
+                          "type": "text"
+                        }
+
+                      ]
+                    },
+                    {
+                      "title": "%arc.postgres.settings.storage.title%",
+                      "fields": [
                         {
                           "label": "%arc.storage-class.data.label%",
                           "description": "%arc.postgres.storage-class.data.description%",
                           "variableName": "AZDATA_NB_VAR_POSTGRES_STORAGE_CLASS_DATA",
                           "type": "kube_storage_class",
                           "required": true
                         },
+                        {
+                          "label": "%arc.postgres.server.group.volume.size.data.label%",
+                          "description": "%arc.postgres.server.group.volume.size.data.description%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_VOLUME_SIZE_DATA",
+                          "type": "text",
+                          "textValidationRequired": true,
+                          "textValidationRegex": "^(\\d+)?$",
+                          "textValidationDescription": "%arc.postgres.server.group.volume.size.data.validation.description%",
+                          "defaultValue": "5"
+                        },
                         {
                           "label": "%arc.storage-class.logs.label%",
                           "description": "%arc.postgres.storage-class.logs.description%",
                           "variableName": "AZDATA_NB_VAR_POSTGRES_STORAGE_CLASS_LOGS",
                           "type": "kube_storage_class",
                           "required": true
                         },
+                        {
+                          "label": "%arc.postgres.server.group.volume.size.logs.label%",
+                          "description": "%arc.postgres.server.group.volume.size.logs.description%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_VOLUME_SIZE_LOGS",
+                          "type": "text",
+                          "textValidationRequired": true,
+                          "textValidationRegex": "^(\\d+)?$",
+                          "textValidationDescription": "%arc.postgres.server.group.volume.size.logs.validation.description%",
```

Comment:
There is not a FieldType "integer". In order to verify the user does not place negative numbers or fractions I set the type to text to validate the input with regex.

Reply:
Can we do:
{
  "type": "number",
  "min": 0
}

Reply:
We can but that would mean fractions are valid inputs.

Reply:
I'd rather we be consistent here and use the number spinner (since we use the spinner in other deployments for similar values). Having regex on that to control the acceptable number values can be a feature we add in later if needed

Reply:
Looked into this and yeah - unfortunately it looks like we don't expose the validation to extensions. Created an enhancement for this : https://github.com/microsoft/azuredatastudio/issues/12257